### DISCUSSION TITLE
tinymce Editor For Laravel

### DISCUSSION THREAD

**Discussion by morisfa2** [awcodes/filament-curator#505] [General]
![image](https://github.com/awcodes/filament-curator/assets/136106565/b74ec635-c592-43b9-a541-b4f9b4f1202b)

Hello , do anyone have any idea to can add/select images in editors with filament-curator Package ?

the package :
use Mohamedsabil83\FilamentFormsTinyeditor\Components\TinyEditor;
https://github.com/mohamedsabil83/filament-forms-tinyeditor

or anyway to use a tynymce editor with filament-curator media manager

**Comment by awcodes** [comment]
To my knowledge Tiptap Editor plugin is the only one that works with Curator. The authors of those plugins would have to provide their own integrations to make it work.

**Comment by morisfa2** [comment]
Great ,
The Tiptap Editor  look like be awesome 
i installed that and now is deploying

so i have some issues with TipTap
i uploaded images in tiptap editor and uploaded well
but also is not visible in media manager menu based in /media

do you have any idea ?



and we required to every time upload again a image
![image](https://github.com/awcodes/filament-curator/assets/136106565/1031ba20-1640-415e-bc58-93d4e397752a)

it is possible to can select from MediaManager/UploadedImages ?



thanks

**Reply by awcodes** [reply]
Sorry, not sure what you are doing? Did you change the Tiptap editor config to use the Media Action from Curator?

**Reply by morisfa2** [reply]
Thanks Worked
why i dont see that commented line for Curator MediaAction 
done,
